docs/cvl/builtin.md

```diff
@@ -1,10 +1,27 @@
 Built-in Rules
 ==============
+`Certora` Prover has built-in general-purpose rules targeted at finding known vulnerabilities.
+These rules can be verified on a contract out-of-the-box.
 
-The Prover has some built-in general-purpose rules that can be verified on a
-contract out-of-the-box.
+You can add several builtin rules and all other rule types in the same spec file.
+
```

Comment:
It would be good to add a contents block here; see the other pages in this directory for examples (e.g. `rules.md`

Reply:
I am not sure what you mean. Can you specify the specific example in rules.md?